edi-tools/modules/codegen/schema_code_gen.bal

```diff
@@ -28,26 +28,47 @@ public function generateCodeForSchema(json schema, string outputPath) returns er
     string schemaCode = string `
 import ballerina/edi;
 
+# Convert EDI string to Ballerina ${ediSchema.name} record.
+# 
+# + ediText - EDI string to be converted
+# + return - Ballerina record
 public isolated function fromEdiString(string ediText) returns ${ediSchema.name}|error {
     edi:EdiSchema ediSchema = check edi:getSchema(schemaJson);
     json dataJson = check edi:fromEdiString(ediText, ediSchema);
     return dataJson.cloneWithType();
 }
 
+# Convert Ballerina ${ediSchema.name} record to EDI string.
+# 
+# + data - Ballerina record to be converted
+# + return - EDI string
 public isolated function toEdiString(${ediSchema.name} data) returns string|error {
     edi:EdiSchema ediSchema = check edi:getSchema(schemaJson);
     return edi:toEdiString(data, ediSchema);    
 } 
 
+# Get the EDI schema.
+# 
+# + return - EDI schema
```

Comment:
Same as other places the error is ignored in docs

Reply:
Changed with 1330f90e144a8514f5a75f7fd1d041a44035e7ba.